README.md

```diff
@@ -170,3 +170,13 @@ Our automated tests running in CI create lot of repositories in our redhat-appst
 There is a mage target that can cleanup those repositories - `mage local:cleanupGithubOrg`.
 
 For more infor & usage, please run `mage -h local:cleanupGithubOrg`.
+
+# PR review process in e2e-tests
+* Both [CODEOWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/.github/CODEOWNERS) file and [OWNERS files](https://github.com/redhat-appstudio/e2e-tests/blob/main/OWNERS) (in a directory or its parent directory) take effect in e2e-tests PR review process.
+  * Code owners are automatically requested for review when open a PR. While openshift-ci bot also request a review according to the set of OWNERS files nearest to the code being changed.
+  * [CODEOWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/.github/CODEOWNERS) and root [OWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/OWNERS) are maintained by the whole RHTAP QE team. While sub directory OWNERS files are maintained by sub QE team.
+
+* To get a PR merged, following is needed:
+  1. Approval via GitHub UI(Review changes -> select Approve -> Submit review). This approval is valid only from repository collaborators (with Triage permissions or higher).
```

Comment:
```suggestion
  1. Approval via GitHub UI (Review changes -> select Approve -> Submit review). This approval is valid only from repository collaborators (with Triage permissions or higher).
```

Reply:
I'm fine with this PR once Simon's comments are fixed